is/esb/newmount.sh

```diff
@@ -29,16 +29,16 @@ sudo apt-get -y install nfs-common
 sudo apt-get -y install nfs-kernel-server
 
 #create the drive that is being mapped to if doesn't exist
-if [ ! -d $4 ]; then
-  echo "missing directory, creating $4"
-  sudo mkdir -p $4
-  sudo chmod 777 $4
+if [ ! -d $3 ]; then
+  echo "missing directory, creating $3"
+  sudo mkdir -p $3
+  sudo chmod 777 $3
 fi
 
 # Map storage account to drive if 4 parameters exist.
```

Comment:
Now that there is only 3 params should read: `Map storage account to drive if 3 parameters exist` or perhaps even better `Map storage account to drive if required parameters exist.` since if you ever change the number of parameters the comment is still correct.

Reply:
Sure, i have changed it